Sources/Core/Sources/Render/RenderCoordinator.swift

```diff
@@ -62,9 +62,7 @@ public class RenderCoordinator: NSObject, RenderCoordinatorProtocol, MTKViewDele
     
     updateCamera(client.game.player, view)
     
-    guard
-      let transparentAndOpaqueCommandBuffer = commandQueue.makeCommandBuffer(),
-      let translucentCommandBuffer = commandQueue.makeCommandBuffer()
+    guard let worldRenderCommandBuffer = commandQueue.makeCommandBuffer()
     else {
```

Comment:
put the else on end the previous line given that the guard is not multiline anymore